modules/rostests/apitests/user32/LoadImageGCC.c

```diff
@@ -0,0 +1,256 @@
+/*
+ * PROJECT:     ReactOS api tests
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Test for LoadImageW using DLL compiled with MSVC
+ * PROGRAMMER:  Doug Lyons <douglyons@douglyons.com>
+ * COPYRIGHT:   Copyright 2024 Doug Lyons <douglyons@douglyons.com>
+ * NOTE:        Works for ReactOS, but not in Windows 2003 Server SP2
+ */
```

Comment:
Please use modern header per coding style.

Reply:
I see nothing wrong here, except that the PROGRAMMER is a redundant field here that should be stripped. Anything more than that? Do you dislike the GPL and recommend LPGL instead?

Reply:
Should be fixed. Thanks.